dist/spdx-tools.cjs

```diff
@@ -3,10 +3,16 @@
 var fs = require('fs/promises');
 var uuid = require('uuid');
 
+// SPDX-FileCopyrightText: 2023 spdx contributors
+//
+// SPDX-License-Identifier: MIT
```

Comment:
why are you adding file headers to every function? This header should just appear once

Reply:
ah, generated files